docs/differenceBetweenPBSGo-and-Java.md

```diff
@@ -1,36 +1,34 @@
 # Differences Between Prebid Server Go and Java
 
-October 26, 2018
+January 24, 2019
 
 The sister Prebid Server projects are both busy and moving forward at different paces on different features. Sometimes a feature may exist in one implementation
 and not the other for an interim period. This page tracks known differences that may persist for longer than a couple of weeks.
 
 ## Feature Differences
 
-1) PBS-Java supports Currency conversion. PBS-Go [issue 280](https://github.com/prebid/prebid-server/issues/280). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
+1) PBS-Java supports Currency conversion. PBS-Go currently has it disabled as it's still under dev [issue 280](https://github.com/prebid/prebid-server/issues/280), [issue 760](https://github.com/prebid/prebid-server/pull/760). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
 1) PBS-Java supports IP-address lookup in certain scenarios around GDPR. See https://github.com/rubicon-project/prebid-server-java/blob/master/docs/developers/PrebidServerJava_GDPR_Requirements.pdf
-1) PBS-Java supports InfluxDB and Graphite, PBS-Go supports InfluxDB and Prometheus as metrics backend.
+1) PBS-Java supports InfluxDB, Graphite and Prometheus, PBS-Go supports InfluxDB and Prometheus as metrics backend.
 1) PBS-Java has Circuit Breaker mechanism for database, http and geolocation requests. This can protect the server in scenarios where an external service becomes unavailable.
 1) PBS-Java supports `ext.prebid.cache.{bids,vastxml}.ttlseconds` field to pass the TTL to Cache Service 
-(per-impression, per-account and per-mediaType options are in progress [PR 154](https://github.com/rubicon-project/prebid-server-java/pull/154)),
+(per-impression option is in progress [PR 154](https://github.com/rubicon-project/prebid-server-java/pull/154)),
```

Comment:
The `per-impression option` is completed also. See `org.prebid.server.cache.CacheService#cacheBidsOpenrtb` as ref.

Reply:
Then I guess it's not a difference anymore, other than "PBS-Go use 60-seconds buffer"

Reply:
yes

Reply:
Actually, removed this one completely as this moment is already mentioned separately in Minor differences (60 sec buffer)